.github/workflows/generate-pr-description.yml

```diff
@@ -0,0 +1,66 @@
+name: Auto PR Description
+
+on:
+  pull_request:
+    types: [opened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  auto-describe:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Comment:
If I open the PR as a draft and then later change it to "Ready for Review", this code will not run again.